pkg/corerp/datamodel/environment.go

```diff
@@ -50,12 +50,17 @@ type EnvironmentProperties struct {
 type RecipeConfigProperties struct {
 	// Terraform specifies the terraform config properties
 	Terraform TerraformConfigProperties `json:"terraform,omitempty"`
+
+	Env EnvironmentVariables `json:"env,omitempty"`
```

Comment:
Should we call this EnvVariables or similar? Env can be mistaken to be environment which is prevalent in codebase (apologies since i wrote the design doc :D)